content/rc/cloud-integrations/prometheus-integration.md

```diff
@@ -0,0 +1,175 @@
+---
+Title: Prometheus integration with Redis Cloud
+linkTitle: Prometheus integration
+description: To collect and display metrics data from your databases, you can connect your Prometheus or Grafana server to your Redis Cloud databases.
```

Comment:
Just to be technically accurate/clear: ... you can connect Prometheus to the 'Prometheus compatible endpoint' in the Redis Cloud subscription, then you can connect Grafana to Prometheus for visualization.  You can't connect Grafana directly to the RC metrics endpoint and you technically don't need Grafana, it's just a very common pattern to add Grafana for visualization.